native/cocos/platform/win32/modules/SystemWindow.cpp

```diff
@@ -32,30 +32,37 @@
 #include "platform/SDLHelper.h"
 #include "platform/win32/WindowsPlatform.h"
 #include "sdl2/SDL_clipboard.h"
+#include "platform/win32/modules/SystemWindowManager.h"
 
 namespace cc {
-SystemWindow::SystemWindow(IEventDispatch *delegate)
-: _sdl(std::make_unique<SDLHelper>(delegate)) {
+SystemWindow::SystemWindow(IEventDispatch *delegate, uint32_t windowId, void *externalHandle)
+    : _windowId(windowId)
+{
+    _sdl = SDLHelper::getInstance();
 }
 
 SystemWindow::~SystemWindow() {
 }
 
 int SystemWindow::init() {
-    return _sdl->init();
+    //return _sdl->init();
+    return 0;
 }
 
 void SystemWindow::pollEvent(bool *quit) {
-    return _sdl->pollEvent(quit);
+    //return _sdl->pollEvent(quit);
```

Comment:
If pollEvent is not supported anymore, shall we remove it completely?

Reply:
Yes，I will remove this function.